### DISCUSSION TITLE
Documentation Edits

### DISCUSSION THREAD

**Discussion by lakbychance** [0xTheProDev/Operational-Transformation#63] [General]
Hi @Progyan1997 , I was going through the [CONTRIBUTING.md](https://github.com/Progyan1997/Operational-Transformation/blob/main/.github/CONTRIBUTING.md) for this project and found subtle grammatical nuances that I think can be corrected. Also the README link in the same redirects to 404 Page. Since the guidelines state that an issue shouldn't be opened up by a user until a maintainer picks it from discussions and these kind of changes don't fall under **bug report**,  can I proceed with creating an issue and submitting a PR that addresses it ? 

BTW, this is a really cool project 💯 . I haven't really worked in the same space so the code is new but interesting. I think a diagrammatic workflow would be good addition to README as well so that new comers can familiarize themselves quickly with terminologies like "Operational Transformation", "Editor Adapters" and "Database Adapters".

**Comment by 0xTheProDev** [comment]
Hi @lapstjup , I have created https://github.com/Progyan1997/Operational-Transformation/issues/64 for you on the task of improving quality of the Documentation the project currently has.
I have a plan to have a formal documentation site with much more detailed explanations along with proper diagram. Or maybe a Wiki page.

**Reply by lakbychance** [reply]
Awesone @Progyan1997 . The corrections are minor. I think you can skip the **Hacktoberfest** label for it so that such changes don't get misinterpreted for **easy** way of completing the same and encourage spamming. What do you think ?

**Reply by 0xTheProDev** [reply]
Don't worry about that. Anyway all PRs for the repo this month will be counted toward Hacktoberfest.

**Reply by 0xTheProDev** [reply]
@lapstjup Your change has been merged to `main` branch.
